samples/net/mqtt_publisher/README.rst

```diff
@@ -97,11 +97,38 @@ of this sample application (i.e., :zephyr_file:`samples/net/mqtt_publisher`) and
    :goals: build flash
    :compact:
 
+If the board is connected directly to the Linux host computer through LAN,
+configure the network interface:
+
+.. code-block:: console
+
+	$ IFACE=eth0 # Change this to the interface to which the LAN cable is connected
+
+	$ IPV4_ADDR_1="192.0.2.2/24"
+	$ IPV4_ROUTE_1="192.0.2.0/24"
+
+	$ sudo ip address add $IPV4_ADDR_1 dev $IFACE
+	$ sudo ip route add $IPV4_ROUTE_1 dev $IFACE
+	$ sudo ip link set dev $IFACE up
```

Comment:
Thanks, this looks much clearer now. Although I almost never use static addresses for external boards connected to my LAN, instead I enable DHCPv4 and use it directly. This way there should be no configuration needs in the host side and things should just work.